src/te/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -418,7 +418,7 @@ Array<Tensor> CacheWriteWithReLayoutTensor(Schedule sch,
   }
 
   Operation cache_op = TensorComputeOpNode::make(
-      tensor_op->name + "." + scope, tensor_op->tag, new_axis,
+      (String)((std::string)tensor_op->name + "." + scope), tensor_op->tag, new_axis,
```

Comment:
Shouldn't we be able to directly constructor a String object from std::string or const char*?

https://github.com/apache/incubator-tvm/blob/f939c61805e77f1f63ec44676c4e936a94055908/include/tvm/runtime/container.h#L368-L375

Reply:
Hi @zhiics ,

Thanks for your kind reminder. I remove the `String()` here.